packages/theme/src/theme.ts

```diff
@@ -107,6 +127,7 @@ export interface Theme {
   icon: {
     size: typeof iconSizes;
   };
+  geofencingZones: GeofencingZonesPaintProps;
```

Comment:
What do you think, should this be renamed to `geofencingZonesPaintProps`, or better kept as just `geofencingZones`?

Reply:
geofencingZones ✌️